### ISSUE TITLE
bug(yarn format): yarn format command not working as expected

### ISSUE THREAD

**Issue by swseverance** [angular/components#19879]
#### Reproduction

Steps to reproduce:
1. Open `src/material/autocomplete/autocomplete.ts`
2. Delete line breaks so that one of the lines is very long (and would get picked up by the formatter). For example:
```typescript
export class MatAutocomplete extends _MatAutocompleteMixinBase implements AfterContentInit, CanDisableRipple, OnDestroy {
```
3. Save the file
4. Run `yarn format`
 
 
#### Expected Behavior

The `yarn format` command should insert linebreaks back into `src/material/autocomplete/autocomplete.ts` so that the file is properly formatted.

#### Actual Behavior

No formatting takes place. I see:
```
yarn run v1.22.0
$ yarn ng-dev format changed
$ /Users/ssevera/Documents/material2/node_modules/.bin/ng-dev format changed
No files matched for formatting.
✨  Done in 2.31s.
```

#### Environment

- Angular:
- CDK/Material:
- Browser(s):
- Operating System (e.g. Windows, macOS, Ubuntu): macOS
- Yarn version: 1.22.0
- Node version: v12.18.0

**Comment by crisbeto** [comment]
I think the `yarn format` command only handles .bazel files right now.

**Comment by devversion** [comment]
Yeah, @crisbeto is right. We haven't set up any JS formatter in this repository as we do not enforce it anywhere. We might want to set clang-format up, but not enforce it. Ideally though we'd enforce formatting for TS files somehow then (incrementally)
